Tasks/sum.js

```diff
@@ -1,20 +1,17 @@
 // Sum all numbers from an array
 
-const k = (sum = function (s = () => {}) {
-  sum = [0];
-  let k = 5;
-  for (i of s) {
-    let t = typeof i;
-    if (t === 'number') {
-      if (sum.length > 0) {
-        const new_Sum = sum[sum.length - 1] + i;
-        sum.push(new_Sum);
-      }
-    }
-  }
-  sum[0];
-  return sum[sum.length - 1];
-  sum.push(sum.length - 1);
-});
+// Step 6(1)
+// Implement with Array.map()
+
+'use strict';
+
+const sum = (array) => {
+  let sum = 0;
```

Comment:
Same name for function and variable will work but it's not of for human readability

Reply:
Reworked that 
https://github.com/M-Rybalko/Exams/tree/exam